Planner/Planner/src/Users.java

```diff
@@ -0,0 +1,47 @@
+import javax.swing.JFrame;
+import javax.swing.JButton;
+import javax.swing.JPanel;
+import javax.swing.JList;
+import javax.swing.JOptionPane;
+
+import java.awt.FlowLayout;
+import java.awt.Frame;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class Users extends JFrame implements ActionListener {
```

Comment:
Please change the name of this class in Planner. This will be the main window of the application.